service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -81,6 +82,12 @@ public static String buildCallout(final CaseData caseData) {
             caseData.getFirstHearing().orElse(null));
     }
 
+    public static String buildCalloutWithNextHearing(final CaseData caseData) {
```

Comment:
instead of having a static method could it be instanced, and then we can autowire time in and use that to compare instead of LocalDateTime.now()

Reply:
actually there is a method that accommodates this already, `buildCalloutWithNextHearing` that accepts `CaseData` and a `LocalDataTime`

Reply:
A small problem with using that function is that it adds a "^" which will result in the email having a "^^" resulting in a double callout due to the template already having a "^"